charts/bookkeeper/Chart.yaml

```diff
@@ -1,12 +1,12 @@
 name: pravega
-version: 0.5.0
-appVersion: 0.5.0
-description: pravega Helm chart for Kubernetes
+version: 0.6.1
+appVersion: 0.6.1
+description: Bookkeeper Helm chart for Kubernetes
 keywords:
 - pravega
 - storage
 home: https://github.com/pravega/pravega
```

Comment:
This and "sources" perhaps needs to change.

Reply:
done